app/pagination.rb

```diff
@@ -0,0 +1,8 @@
+class Pagination
+  attr_reader :start, :count
+
+  def initialize(options = {})
+    @start = options[:start] ? options[:start].to_i : 0
+    @count = options[:count] ? options[:count].to_i : 50
+  end
+end
```

Comment:
Currently, this Pagination class holds the state for pagination. I wonder if it could also hold it's behaviour?

e.g.

```ruby
paginator = Paginator.new(page_size: 50)

items = paginator.paginate(DraftContentItem.all, page: 3)
#=> ActiveRecord::Relation

items.count
#=> 50
```

What do you think?